src/SASjs.ts

```diff
@@ -167,6 +167,20 @@ export default class SASjs {
     return await this.sasViyaApiClient!.deleteContext(contextName, accessToken)
   }
 
+  /**
+   * Returns a compute context.
+   * @param contextName - the name of the context to return.
```

Comment:
same comment as above.  Also given that this has no parallel in SAS 9 - perhaps we should be more explicit, eg:

`getViyaComputeContextByName()`